### PR TITLE
Added GUI for builder to display list of enabled environments from ym…

### PR DESCRIPTION
**Pull request by iguannalin**
…l file

## Description

GUI for builder plugin

## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

n/a

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code

### CODE REVIEW

File: hook.py [hunk 1]
```diff
@@ -1,17 +1,24 @@
...
 access = BaseWorld.Access.RED
 
 
 async def enable(services):
+    environments = BaseWorld.strip_yml('plugins/builder/conf/environments.yml')
     BaseWorld.apply_config('build', BaseWorld.strip_yml('plugins/builder/conf/environments.yml')[0])
     build_svc = BuildService(services)
     await build_svc.stage_enabled_dockers()
 
+    envs = environments[0]['enabled']
```
Comment (iguannalin) [inline hook.py]: This assumes that there's always an `'enabled'` object in the `environments.yml` file, should I add in a check for this and make it an empty list if `'enabled'` doesn't exist?

**Review by nopfor — CHANGES_REQUESTED** [review]
Could the UI just list the "Currently supported build languages" (ex: csharp, cpp_windows_x64)? The other information isn't necessary, but the build languages might help people build abilities.

**Review by iguannalin — COMMENTED** [review]
> Could the UI just list the "Currently supported build languages" (ex: csharp, cpp_windows_x64)? The other information isn't necessary, but the build languages might help people build abilities.

Would it be helpful to add `"Currently supported build languages"` as a label for that list as well?

File: hook.py (hunk 1 above)
Comment (nopfor) [reply hook.py]: Oh yeah, probably!

**Comment by nopfor** [comment]
> > Could the UI just list the "Currently supported build languages" (ex: csharp, cpp_windows_x64)? The other information isn't necessary, but the build languages might help people build abilities.
> 
> Would it be helpful to add `"Currently supported build languages"` as a label for that list as well?

Yeah, a label would be great.

**Review by nopfor — APPROVED** [review]
LGTM!

**Pull request merged**